pkgs/purs/src/Wallet/PrivateKey.purs

```diff
@@ -140,3 +145,16 @@ foreign import mnemonicToEntropyImpl :: String -> String
 foreign import privKeyToAddressImpl :: String -> String
 
 foreign import addressToNonceImpl :: String -> BigInt
+
+foreign import isPrivateKeyImpl :: String -> Boolean
+
+--------------------------------------------------------------------------------
+-- Instances
+--------------------------------------------------------------------------------
+instance decodeJsonPrivateKey :: DecodeJson PrivateKey where
+  decodeJson j = do
+    s <- decodeJson j
```

Comment:
It think it would be important to check the string with `isPrivateKeyImpl` to make the instance correct.